Documentation/build-seissol.rst

```diff
@@ -0,0 +1,190 @@
+..
+  SPDX-FileCopyrightText: 2022-2024 SeisSol Group
+
+  SPDX-License-Identifier: BSD-3-Clause
+
+.. _build_seissol:
+
+Compiling and Running SeisSol
+-----------------------------
+
+For this page, we will assume that all the environment variables
+from the previous section to be loaded (cf. section ). That is, you have your SeisSol build directory in ``$SEISSOL_BASE``.
+
+Get the latest version of SeisSol on git by cloning the repository,
+including all submodules:
+
+.. code-block:: bash
+
+   git clone --recursive --branch vX.Y.Z --depth 1 https://github.com/SeisSol/SeisSol.git
+
+You may also do a shallow clone by adding ``--depth=1`` which will save a bit of download volume.
+
+To download the full repository, as needed e.g. for development, run
+
+.. code-block:: bash
+
+   git clone --recursive https://github.com/SeisSol/SeisSol.git
+
+Note, however, that the master branch may be unstable.
+
+Building SeisSol
+~~~~~~~~~~~~~~~~
+Then, given that you have built or installed all required dependencies, you may compile SeisSol for CPUs as follows (or use ``ccmake`` for a GUI):
+
+.. code-block:: bash
+
+   mkdir -p build && cd build
+   cmake -DNUMA_AWARE_PINNING=ON -DASAGI=ON -DPRECISION=double -DORDER=4 -DEQUATIONS=elastic ..
+   make -j 4
+
+Note that you may need to adjust your host architecture, especially if you are on an ARM-based machine: then you will need to add at least ``-DHOST_ARCH=neon``, or one of the other ARM-compatible architectures.
+Sometimes, the ninja build tool can speed up your build process. To use it, add ``-GNinja`` to the cmake command (you might need to delete your cmake folder first to switch from make).
+
+Generally, we recommend using ``PRECISION=single`` for faster simulations. If you experience Inf/NaN errors (cf. also https://github.com/SeisSol/SeisSol/issues/200 ), then setting ``PRECISION=double`` is a good idea.
+The ``ORDER`` equals the degree of the used polynomials plus one. In general, we optimize for the orders 3 to 6 the most.
```

Comment:
"In general, we optimize for the orders 3 to 6 the most"
why?
Maybe remove statement